torch/_refs/__init__.py

```diff
@@ -1319,6 +1320,15 @@ def _trunc_divide(
 #
 
 
+@out_wrapper
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("self", "tensor1", "tensor2"),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
+)
+def addcdiv(self: Tensor, tensor1: Tensor, tensor2: Tensor, value: float = 1):
+    return self + value * (tensor1 / tensor2)
```

Comment:
We may need some logic here similar to the logic for `alpha` in `aten.add` when it comes to castings.

Reply:
Also, how does this implementation relate to the `warning` in the docs https://pytorch.org/docs/stable/generated/torch.addcdiv.html? Does this implementation work as expected for integer tensors? Could you add some reference inputs to the OpInfo (note that this is not the same as the sample inputs) to test for this?